### PR TITLE
[Core] Adding CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**
This was missing.

**🆕 Changelog**
- Adding CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+If you are already a developer, please create a new branch with the proposed changes. Then create a **Pull Request** (**PR**) and request a review. Define the proper labels in the corresponding **PR**.
+
+We consider the same branch structure: `subject/short-description`. For example for a fix in the core of a given utility: `core/fix-xxx-utility`. 
+
+If you are not developer, fork this project and open a **PR** with the proposed fix. If you want to become a developer, please check next section.
+
+## Becoming a maintainer 👨‍💻
+
+Please contact the main maintainer at <kratosmultiphysics@gmail.com> 📨.
```
Comment (rubenzorrilla) [inline CONTRIBUTING.md]:
```suggestion
Please contact the main maintainers at <kratosmultiphysics@gmail.com> 📨.
```

**Review by rubenzorrilla — COMMENTED** [review]
Just mentioning that I like it. Thanks Tote!

**Comment by loumalouomega** [comment]
> Pinging @antonialarese as she explicitly asked me for this a few days ago. As usual @loumalouomega took the lead smile

As always, sync, haha

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+The structure of the branch must be: `subject/short-description`. For example for a new utility in the core: `core/adding-xxx-utility`. We separate the words using `-`.
+
+## Submitting a hotfix 🔥
+
+If you are already a developer, please create a new branch with the proposed changes. Then create a **Pull Request** (**PR**) and request a review. Define the proper labels in the corresponding **PR**.
+
+We consider the same branch structure: `subject/short-description`. For example for a fix in the core of a given utility: `core/fix-xxx-utility`. 
+
+If you are not developer, fork this project and open a **PR** with the proposed fix. If you want to become a developer, please check next section.
+
```
Comment (pooyan-dadvand) [inline CONTRIBUTING.md]: If you are not member of the KratosMultiphysics organization you may fork this project and open a **PR** with the proposed fix. On a side note, you are welcome to become a member of our organization and for joining us please see below

**Review by roigcarlo — APPROVED** [review]

**Comment by roigcarlo** [comment]
And what is that?

**Comment by loumalouomega** [comment]
https://github.blog/changelog/2020-06-23-community-content-reports-included-in-community-profile/

**Comment by roigcarlo** [comment]
I see. Well, afaik triangles and boundary conditions can hardly be considered targets for content report

For me having the code of conduct in our case is more than enough, but if you want to enable it...

**Pull request merged**